src/System.Text.Utf8/src/System/Text/Utf8/Utf8EncodedCodePoint.cs

```diff
@@ -21,7 +21,7 @@ public unsafe Utf8EncodedCodePoint(char character) : this()
 
             fixed (byte* encodedData = &_byte0)
             {
-                ByteSpan buffer = new ByteSpan(encodedData, 4);
+                Span<byte> buffer = new Span<byte>(encodedData, 4);
```

Comment:
I'm not a big fan of vars - I usually spend more trying to figure out what var means than writing 20 more characters...